crates/holochain_sqlite/src/sql/conductor/is_blocked.sql

```diff
@@ -0,0 +1,9 @@
+-- count rows with a block for any reason against the target
+SELECT
+  COUNT(1) > 0
```

Comment:
what is this magic? wouldn't this just always be `true`?

Reply:
@maackle count is an aggregation function, sql doesn't have real bool types, bool is just an alias for integer, so this is something like `count(true)` as in "count every row", but without it needing to load any data to check for null values.

the `> 0` is a normal inequality, we want to know if there's more than 0 rows.